.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
```

Comment:
For this part, I did not know whether I should modify part or all of this. I added a title inside the single quotes on the 'Title' line, but it is redundant with the Title of the issue

Reply:
name: is the name of the issue template as it appears to the user after clicking the Issues tab: [ref this](https://help.github.com/en/articles/about-issue-and-pull-request-templates)
title: is both the placeholder text and default issue title when the user is filling out the issue form.
title is mutable by the user, name is not.
title is optional, and I really didn't know what a god default issue title might be. I figure using a square bracket tag might help with sorting, but the user can always just delete it when they create the issue.

label: is the important bit, this is the label that gets auto-assigned to the new issue and the is the hint the sorter uses to route the new issue to the correct project. The questions project is a bit overkill, but having it means that the actual bug tracker is less-likely to get polluted with questions. (although we could just manually curate this, i have no idea what volume to expect)